app/helpers/address_helper.rb

```diff
@@ -123,4 +123,19 @@ def compare(ordnance_address, edited_address)
 
     sanitized_edited_address & sanitized_ordnance_address == sanitized_ordnance_address
   end
+
+  def remove_selected_address_changes(selected_address)
```

Comment:
```suggestion
  def remove_changes_to_ordnance_survey_api_response(ordnance_address)
```

Reply:
Thanks @leenagupte - like the name.  Updated.